### DISCUSSION TITLE
Move feed items sources to new feed

### DISCUSSION THREAD

**Discussion by jgritten** [dailydotdev/daily#1506] [Feature Requests]
I would like the ability to send certain sources to a Custom Feed. Currently, through the process of my daily feed viewing, it has curated many Comics (eg. Work Chronicles) to populate the top. I like comics but when I want news, I dont want to see comics. So the feature I would like is from the triple dots context menu, add a menu item called "Send to feed...", then within a submenu, select which of your custom feeds you'd like to send it to. This would effectively filter posts from Work Chronicles on "My Feed", and add a follow to my targeted custom feed. 

This way, whenever I want to see comics, I can simply goto my comics feed, while leaving me comic free in my news feed.

thanks and let me know if you have any questions

**Comment by idoshamun** [comment]
@jgritten that's a great suggestion. we will consider it
